chemicaltoolbox/datatypes/molecules.py

```diff
@@ -449,6 +449,33 @@ def set_peek( self, dataset, is_multi_byte=False ):
             dataset.peek = 'file does not exist'
             dataset.blurb = 'file purged from disk'
 
+class PDBQT(GenericMolFile):
+    """
+    Autodock Format
+    The PDBQT format stores the atomic coordinates, partial charges and AutoDock atom types,
+    for both the receptor and the ligand.
+    http://autodock.scripps.edu/faqs-help/faq/what-is-the-format-of-a-pdbqt-file
+    """
+    file_ext = "pdbqt"
+
+    def sniff(self, filename):
+        if count_special_lines('COMPND', filename) > 0:

```

Comment:
This file format is pretty similar to pdb file format, in cases I've been contact the head of file begins with one or other (COMPND or ATOM), I implemented this way because of it. Are you in agreement? If you have a better suggestion would be great! :smile: 
If you are in according I can do this work and commit in molecules.py.

```Galaxy now supports all chemical datatypes natively!! :)```

AWESOME!

Thanks for your help.